### PR TITLE
Switch to zinit-continuum.

### PR DESCRIPTION
**Pull request by carrete**
I'm new to zsh, so I'm unfamiliar with this history. I'm making the switch to
zsh after having recently received a new MacBook Pro, and learning that zsh is
now the new default shell on macOS. The zdharma/zplugin repo no longer exists.
In researching this issue, I came across this [post on reddit][1]. zdharma
deleted the zplugin repo and others. The successor to zplugin appears to be
[zinit-continuum][2]. (Looks like at some point zplugin was renamed to zinit?)

zinit is "installed" by `sh -c "$(curl -fsSL https://git.io/zinit-install)"`.
The result of this step is simply the section I've added to .zshrc in this
pull-request. This is what really installs zinit. Given that radian depends on
zinit, it seems reasonable to me that it should be included in radian.
Otherhwise, the alternative is to have users install zinit on their own, and
move its installation block from .zshrc to .zshrc.local? This seems a bit
klunky to me.

This probably also obsoletes the need to check for various installation paths,
and means that current users of radian will be "upgraded" to zinit-continuum
the next time they update their installation of radian. I don't know how
impactful this will be, nor the best way to roll this out. I'm happy to adjust
this pull-request accordingly. Thanks

[1]https://www.reddit.com/r/zsh/comments/qinb6j/httpsgithubcomzdharma_has_suddenly_disappeared_i/
[2]https://github.com/zdharma-continuum/zinit/wiki/🚚-Migration-to-zdharma-continuum

### CODE REVIEW

File: shell/zsh/.zshrc [hunk 1]
```diff
@@ -24,6 +47,8 @@ elif [[ -f /usr/share/zsh/plugin-managers/zplugin/zplugin.zsh ]]; then
     radian_zplugin="/usr/share/zsh/plugin-managers/zplugin/zplugin.zsh"
 elif [[ -f ~/.zplugin/bin/zplugin.zsh ]]; then
     radian_zplugin="$HOME/.zplugin/bin/zplugin.zsh"
+elif [[ -f ~/.local/share/zinit/zinit.git/zinit.zsh ]]; then
+    radian_zplugin="$HOME/.local/share/zinit/zinit.git/zinit.zsh"
```
Comment (raxod502) [inline shell/zsh/.zshrc:51]: I agree with you that we can get rid of these other cases most likely.

**Review by raxod502 — COMMENTED** [review]
Thanks! I can test this out and merge.

**Comment by raxod502** [comment]
Read through that Reddit thread you linked, that whole affair seems quite bizarre, but yes, looks like zdharma-continuum is the way to go moving forward.

**Comment by raxod502** [comment]
CI issue is unrelated. Working on fixing that now.

**Comment by raxod502** [comment]
Fixed CI in https://github.com/raxod502/radian/pull/499.

**Comment by carrete** [comment]
Cool. I like the changes you made 👍

**Pull request merged**